src/registry/Commands.chpl

```diff
@@ -280,21 +577,35 @@ proc ark_repeatFlat_bool_1(cmd: string, msgArgs: borrowed MessageArgs, st: borro
   return ManipulationMsg.repeatFlatMsg(cmd, msgArgs, st, array_dtype=bool, array_nd=1);
 registerFunction('repeatFlat<bool,1>', ark_repeatFlat_bool_1, 'ManipulationMsg', 903);
 
-import OperatorMsg;
-
-import ParquetMsg;
```

Comment:
It isn't clear to me why we can get rid of all these imports in this file now and I'm not seeing where that is correspondingly changed in the Python scripts. What am I missing here?

Reply:
These can be removed because those modules don't have any `registerCommand` or `instantiateAndRegister` annotations yet. This was mostly fixed by: https://github.com/Bears-R-Us/arkouda/pull/3371/commits/f08fdf6d436871e845e06081c8b54ad4c21ffcae (in the previous PR), but didn't take affect because I also needed the change on line `619` of `register_commands.py` (added in this PR).